src/screens/Home/index.js

```diff
@@ -6,7 +6,7 @@ import Header from 'screens/Home/header';
 
 const Home = () => {
   return (
-    <PageBackground>
+    <PageBackground type='default'>
```

Comment:
we need a consistency of single and double qoute here. 
```suggestion
    <PageBackground type="default">
```